editor/store/store.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import refx from 'refx';
+import multi from 'redux-multi';
+import { get, flowRight } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { dispatch, getState, subscribe } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { mobileMiddleware } from '../utils/mobile';
+import effects from './effects';
+
+/**
+ * This function applies the custom middlewares used specifically in the editor moodule
```

Comment:
`Applies the custom middlewares used specifically in the editor module`. (Language and typo in module)